src/cmd-buildextend-vmware

```diff
@@ -43,8 +43,8 @@ os.mkdir(tmpdir)
 
 
 def generate_ovf_parameters(qemu, vmdk, cpu=2,
-                            memory=4096, system_type="vmx-07 vmx-08",
-                            os_type="other26xLinux64Guest", scsi="VirtualSCSI",
+                            memory=4096, system_type="vmx-13",
+                            os_type="rhel7_64Guest", scsi="VirtualSCSI",
```

Comment:
Is this the right value for non-RHEL based systems (i.e. Fedora)

Reply:
Great point. In my haste to get this through I did not check on that.